packages/react-native-web/src/exports/Image/index.js

```diff
@@ -186,33 +212,25 @@ const Image: React.AbstractComponent<
     }
   }
 
-  const [state, updateState] = React.useState(() => {
-    const uri = resolveAssetUri(source);
-    if (uri != null) {
-      const isLoaded = ImageLoader.has(uri);
-      if (isLoaded) {
-        return LOADED;
-      }
-    }
-    return IDLE;
-  });
-
+  // Don't raise load events for the fallback source
```

Comment:
Can you please add to this comment to explain _why_ those events shouldn't be triggered (is that what you mean by "raise"?).